region/maridia/inner-green/West Sand Hall.json

```diff
@@ -1374,6 +1374,7 @@
       "flashSuitChecked": true,
       "note": [
         "Use the projectile from a fully submerged Evir to boost Samus into the sandfall, providing just enough height to make the first and hardest jump.",
+        "With reserves, if using auto mode instead of pause abuse to tank the hit, set reserves to 1 energy to use maximum knockback frames; 2 would already make it more difficult.",
```

Comment:
I'd like to see this moved to the detailNote and to be shortened.  Most practically, it should say to use Reserves set to manual and not auto.  But it could be interesting to mention 1 energy of auto is possible.
Setting reserves to exactly 1 is a high level tech too.

The tech requirements aren't impactful here but need to be looked at projectwide for other damageboosts.